diff --git a/models/insulina/tipos.ts b/models/insulina/tipos.ts
index 9a386c0..74dd73a 100644
--- a/models/insulina/tipos.ts
+++ b/models/insulina/tipos.ts
@@ -63,6 +63,7 @@ export type TipoRecomendacao =
   | "AFERIR_JEJUM_3X_SEMANA_15_DIAS"
   | "REPETIR_HBA1C_3_MESES"
   | "REPETIR_HBA1C_6_MESES"
+  | "DOSAR_HBA1C"
   | "AFERIR_PRE_PRANDIAIS"
   | "AFERIR_POS_PRANDIAL"
   | "AVALIAR_ENCAMINHAMENTO_ENDOCRINO"
